system/datatypes/peer_reviewed_article.php

```diff
@@ -1462,6 +1463,17 @@ protected function createCommentDomElement(HyphaDomElement $comment, $review = f
 			return '<span><a href="' . $url . '" target="_blank">' . $url . '</a>' . $suffix.'</span>';
 		}, $commentHtml);
 		$commentHtml = nl2br($commentHtml);
+		if(strlen($commentHtml) > self::MAX_COMMENT_LENGTH){
+			$id = rand();
+			$stub = substr(strip_tags($commentHtml),0, self::MAX_COMMENT_LENGTH*0.7);
```

Comment:
This probably should not hardcode 0.7, but just use a second constant?

Reply:
Done, thanks :)

Reply:
Looks good!